sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileBasedSink.java

```diff
@@ -948,7 +950,12 @@ public final void open(String uId) throws Exception {
           getWriteOperation().getSink().writableByteChannelFactory;
       // The factory may force a MIME type or it may return null, indicating to use the sink's MIME.
       String channelMimeType = firstNonNull(factory.getMimeType(), mimeType);
-      WritableByteChannel tempChannel = FileSystems.create(outputFile, channelMimeType);
+      CreateOptions createOptions =
+          StandardCreateOptions.builder()
+              .setMimeType(channelMimeType)
+              .setExpectFileToNotExist(true)
```

Comment:
Please explain with a comment why this would be true for all file systems.

Reply:
The file is based upon a uuid and thus is unique.  It must also be unique for correctness.
Added comment.